<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>664</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Simple Document Management System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main_simpledms.qrc">
    <normaloff>:/icons/icons/outline_description_white_18dp.png</normaloff>:/icons/icons/outline_description_white_18dp.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_pdfrename">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_monitordir">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Monitor Directory</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_outputfilename">
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Filename</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_outputfilename">
          <property name="toolTip">
           <string>Format: YYYY-MM-DD Documenttitle</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="5">
         <widget class="QListView" name="listView_monitorfiles"/>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_addDate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>FÃ¼ge Datum von heute ein</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="main_simpledms.qrc">
            <normaloff>:/icons/icons/outline_calendar_today_white_18dp.png</normaloff>:/icons/icons/outline_calendar_today_white_18dp.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QTextEdit" name="textEdit_tags"/>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QTreeView" name="treeView_output"/>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QPushButton" name="pushButton_ok">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="main_simpledms.qrc">
            <normaloff>:/icons/icons/outline_check_white_18dp.png</normaloff>:/icons/icons/outline_check_white_18dp.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Tags:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="5">
         <widget class="QLabel" name="thumbnail">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>500</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_settings">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_pref">
           <attribute name="title">
            <string>Base Settings</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_setmonitorfolder">
              <property name="text">
               <string>Monitor Directory</string>
              </property>
              <property name="icon">
               <iconset resource="main_simpledms.qrc">
                <normaloff>:/icons/icons/outline_search_white_18dp.png</normaloff>:/icons/icons/outline_search_white_18dp.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_monitorfolder">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_setdmsroot">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Archive directory</string>
              </property>
              <property name="icon">
               <iconset resource="main_simpledms.qrc">
                <normaloff>:/icons/icons/outline_archive_white_18dp.png</normaloff>:/icons/icons/outline_archive_white_18dp.png</iconset>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_dmsroot">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_rules">
           <attribute name="title">
            <string>Rules</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton_addrule">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="main_simpledms.qrc">
                <normaloff>:/icons/icons/outline_add_circle_outline_white_18dp.png</normaloff>:/icons/icons/outline_add_circle_outline_white_18dp.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QTreeView" name="treeView_rules">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton_deleterule">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="main_simpledms.qrc">
                <normaloff>:/icons/icons/outline_remove_circle_outline_white_18dp.png</normaloff>:/icons/icons/outline_remove_circle_outline_white_18dp.png</iconset>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" rowspan="2" colspan="2">
             <widget class="QTreeView" name="treeView_rulesfolders">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wenn leer, bitte Ordnerstruktur im Archivpfad des Aktenschranks anpassen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16666666</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPdf"/>
   <addaction name="actionSettings"/>
   <addaction name="actionScan"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionScan">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="main_simpledms.qrc">
     <normaloff>:/icons/icons/outline_scanner_white_18dp.png</normaloff>:/icons/icons/outline_scanner_white_18dp.png</iconset>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
   <property name="toolTip">
    <string>Start script to scan document.</string>
   </property>
  </action>
  <action name="actionPdf">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="main_simpledms.qrc">
     <normaloff>:/icons/icons/outline_archive_white_18dp.png</normaloff>:/icons/icons/outline_archive_white_18dp.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="iconText">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Rename pdf and sort into corresponding directory.</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="main_simpledms.qrc">
     <normaloff>:/icons/icons/outline_settings_white_18dp.png</normaloff>:/icons/icons/outline_settings_white_18dp.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="main_simpledms.qrc">
     <normaloff>:/icons/icons/outline_exit_to_app_white_18dp.png</normaloff>:/icons/icons/outline_exit_to_app_white_18dp.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="main_simpledms.qrc">
     <normaloff>:/icons/icons/outline_info_white_18dp.png</normaloff>:/icons/icons/outline_info_white_18dp.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main_simpledms.qrc"/>
 </resources>
 <connections/>
</ui>
